src/LocalizationWrapper.jsx

```diff
@@ -1,33 +1,36 @@
-import React, {Component, PropTypes} from 'react';
+import React, { Component, PropTypes } from 'react';
 import get from 'lodash.get';
 import defaultLocalizer from './util/localize-formatter';
 
 export default (ComposedComponent, messages = {}, customLocalizer) => {
   class ConfigureLocalization extends Component {
-    constructor(props) {
-      super(props);
+    static localize(key, values) {
```

Comment:
I might be missing something obvious, but I'm not seeing how this is used. It also doesn't seem to be documented.